src/main/java/aRC/UI.java

```diff
@@ -0,0 +1,41 @@
+package arc;
+
+import java.util.Scanner;
+
+/**
+ * Encapsulates a UI object that handles user interface
+ */
+public class UI {
+    /**
+     * Prints hello message to the screen
+     */
+    public void hello() {
```

Comment:
Maybe you can consider naming this displayHelloMessage() instead for added clarity 😄